onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -0,0 +1,253 @@
+    base_test_name: str,
+    skip_or_xfails: Iterable[DecorateMeta],
+):
+    """Decorates OpInfo tests with decorators based on the skip_or_xfails list."""
+    ops_mapping = {(info.name, info.variant_test_name): info for info in all_opinfos}
+    for decorate_meta in skip_or_xfails:
+        opinfo = ops_mapping.get((decorate_meta.op_name, decorate_meta.variant_name))
+        assert (
+            opinfo is not None
+        ), f"Couldn't find OpInfo for {decorate_meta}. Did you need to specify variant_name?"
+        decorators = list(opinfo.decorators)
+        new_decorator = opinfo_core.DecorateInfo(
+            decorate_meta.decorator,
+            test_class_name,
+            base_test_name,
+            dtypes=decorate_meta.dtypes,
+        )
+        decorators.append(new_decorator)
+        opinfo.decorators = tuple(decorators)
+
+    # This decorator doesn't modify fn in any way
+    def wrapped(fn):
+        return fn
+
+    return wrapped
+
+
+# Modify this section ##########################################################
+
+# Ops to be tested for numerical consistency between onnx and pytorch
+OPINFO_FUNCTION_MAPPING = {
+    "nn.functional.elu": nn_ops.aten_elu,
+    "nn.functional.relu6": nn_ops.aten_relu6,
+    "nn.functional.selu": core_ops.aten_selu,
+}
+
+TESTED_OPS = frozenset(OPINFO_FUNCTION_MAPPING)
+
+EXPECTED_SKIPS_OR_FAILS = (
+    xfail(
+        "nn.functional.elu",
+        dtypes=dtypes_except(torch.float16, torch.float32),
+        reason="ONNX Runtime doesn't support float64 for Elu",
+    ),
+    xfail(
+        "nn.functional.relu6",
+        dtypes=dtypes_except(torch.float16, torch.float32),
+        reason="ONNX Runtime doesn't support float64 for Relu",
+    ),
+    xfail(
+        "nn.functional.selu",
+        dtypes=dtypes_except(torch.float16, torch.float32),
+        reason="ONNX Runtime doesn't support float64 for Selu",
+    ),
+)
+# END OF SECTION TO MODIFY #####################################################
+
+
+OPS_DB = copy.deepcopy(common_methods_invocations.op_db)
+
+
+class TestOutputConsistency(unittest.TestCase):
+    """Test output consistency between exported ONNX models and PyTorch eager mode.
+
+    This is a parameterized test suite.
+    """
+
+    def setUp(self) -> None:
+        torch.manual_seed(42)
+        np.random.seed(42)
+
+    @common_device_type.ops(
+        [info for info in OPS_DB if info.name in TESTED_OPS],
+        allowed_dtypes=SUPPORTED_DTYPES,
+    )
+    @add_decorate_info(
+        OPS_DB,
+        "TestOutputConsistency",
+        "test_output_match",
+        skip_or_xfails=EXPECTED_SKIPS_OR_FAILS,
+    )
+    def test_output_match(self, device: str, dtype: torch.dtype, op):
+        """Base test method for testing each opset, used by instantiate_device_type_tests."""
+        # device is provided by instantiate_device_type_tests, but we only want to run in cpu.
+        assert device == "cpu"
+
+        samples = op.sample_inputs(
```

Comment:
Why isn't there "shape" information for a sample input?

Reply:
The shape is not explicitly used in this test because onnxscript can handle it in its evaluator. Any considerations?